src/backend/api/Fusion.Resources.Api/Controllers/Personnel/InternalPersonnelController.cs

```diff
@@ -258,7 +258,7 @@ public async Task<ActionResult> ResetAllocationState(string fullDepartmentString
         }
 
         [HttpGet("/projects/{projectIdentifier}/resources/persons")]
-        public async Task<ActionResult> Search([FromRoute] PathProjectIdentifier projectIdentifier, [FromQuery(Name ="$search")] string search, [FromQuery] string? departmentFilter)
+        public async Task<ActionResult> Search([FromRoute] PathProjectIdentifier projectIdentifier, [FromQuery] ODataQueryParams query, [FromQuery] string? departmentFilter)
```

Comment:
what about the "department" argument.
Could you just pass it using odata filter ?
like
.../persons?$search=kjell&$filter=department eq 'whatever' ?